sormas-app/app/src/main/java/de/symeda/sormas/app/util/JurisdictionHelper.java

```diff
@@ -198,6 +200,41 @@ public static EventParticipantJurisdictionDto createEventParticipantJurisdiction
 		return jurisdiction;
 	}
 
+	public static EnvironmentJurisdictionDto createEnvironmentJurisdictionDto(Environment environment) {
+		if (environment == null) {
+			return null;
+		}
+
+		Location environmentLocation = environment.getLocation();
+		if (environmentLocation == null) {
+			return null;
```

Comment:
should not return null here

reporting and responsible users don't depend on the location

Reply:
updated the logic